Project.toml

```
@@ -12,7 +13,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 [compat]
 ChainRulesCore = "0.6.1"
 ChainRulesTestUtils = "0.1.3"
-FiniteDifferences = "^0.7"
+FiniteDifferences = "0.7"
 Reexport = "0.2"
 Requires = "0.5.2, 1"
 julia = "^1.0"
```

since we're doing this for FiniteDifferences

```suggestion
julia = "1"
```